src/main/java/io/cdap/plugin/http/sink/batch/HTTPSinkConfig.java

```diff
@@ -119,11 +135,32 @@ public class HTTPSinkConfig extends BaseHttpConfig {
   @Macro
   private final Boolean disableSSLValidation;
 
-  @Name(NUM_RETRIES)
-  @Description("The number of times the request should be retried if the request fails. Defaults to 3. " +
-    "(Macro enabled)")
+  @Nullable
+  @Name(PROPERTY_HTTP_ERROR_HANDLING)
+  @Description("Defines the error handling strategy to use for certain HTTP response codes." +
+    "The left column contains a regular expression for HTTP status code. The right column contains an action which" +
+    "is done in case of match. If HTTP status code matches multiple regular expressions, " +
+    "the first specified in mapping is matched.")
+  protected String httpErrorsHandling;
+
+  @Name(PROPERTY_ERROR_HANDLING)
+  @Description("Error handling strategy to use when the HTTP response cannot be transformed to an output record.")
+  protected String errorHandling;
+
+  @Name(PROPERTY_RETRY_POLICY)
+  @Description("Policy used to calculate delay between retries.")
+  protected String retryPolicy;
+
+  @Nullable
+  @Name(PROPERTY_LINEAR_RETRY_INTERVAL)
+  @Description("Interval between retries. Is only used if retry policy is \"linear\".")
```

Comment:
Let's add unit of time to avoid confusion.

Reply:
Added. 4e4ee6211be03c9e368a811866c8d96ec49518d3